### PR TITLE
add mass select for kubernetes resources

### PR DESCRIPTION
**Pull request by grosser**
screen shot 2018-06-15 at 5 15 24 pm

@zendesk/compute

### CODE REVIEW

**Comment by adammw** [comment]
can you make it a different colour so it's obvious?

**Comment by grosser** [comment]
color looks kinda gimmicky ... how about no role so it stands out:

screen shot 2018-06-15 at 5 29 43 pm

File: plugins/kubernetes/app/views/kubernetes/deploy_group_roles/edit_many.html.erb [hunk 1]
```diff
@@ -32,17 +33,20 @@
...
           <%= fields_for "kubernetes_deploy_group_roles[#{deploy_group_role.id}]", deploy_group_role do |field| %>
             <% rows.each do |attribute, _, type| %>
-              <td>
-                <% if type == :boolean %>
-                  <%= field.check_box attribute %>
-                <% else %>
-                  <%= field.text_field attribute, size: 10 %>
-                <% end %>
-              </td>
+              <td><%= row_builder.(field, type, attribute) %></td>
```
Comment (adammw) [inline plugins/kubernetes/app/views/kubernetes/deploy_group_roles/edit_many.html.erb]: why do you need the dot? is that the same as .call?

**Comment by grosser** [comment]
yes

**Pull request merged**